dev/Common/CornerRadiusFilterConverters.idl

```diff
@@ -12,6 +12,15 @@ runtimeclass CornerRadiusFilterConverter : Windows.UI.Xaml.DependencyObject, Win
     CornerRadiusFilterKind Filter{ get; set; };
 
     static Windows.UI.Xaml.DependencyProperty FilterProperty{ get; };
+
+    [WUXC_VERSION_MUXONLY]
+    [interface_name("ICornerRadiusFilterConverter2")]
```

Comment:
We only need to do this if we plan to cherry-pick back to and do another 2.2 release. I'm wondering if we're close enough to 2.3 that we shouldn't bother. It's cool to see that we *can* do ABI-compatible changes easily but I wonder if we should.

Oh also because there is a "static" property here this block also needs a [statics_name(...)] attribute to put the ScaleProperty methods on a new static interface too (otherwise the factory's static method layout changes).

Reply:
Yeah we probably shouldn't bother making another 2.2 release at this point, I'll remove the annotation.